src/main/java/com/iota/iri/service/snapshot/Snapshot.java

```diff
@@ -49,9 +47,8 @@
      * track of the milestones that were previously skipped.
      *
      * @param skippedMilestoneIndex index of the milestone that was skipped while applying the ledger state
-     * @return true if the skipped milestone was added to the internal list and false if it was present already
      */
-    boolean addSkippedMilestone(int skippedMilestoneIndex);
+    void addSkippedMilestone(int skippedMilestoneIndex);
```

Comment:
I didn't understand why you changed it. Could be useful (even if you are not using it now), and it doesn't cost really anything. In general I would say that if you can think of something to put instead of `void` then do it.

Reply:
intellij complained about "return type never used" 🗡 i changed it back again